README.md

```diff
@@ -2,7 +2,7 @@
 
 # The Swift Package Index
 
-Adding a new package to the Swift Package Index is straightforward. Every package indexed by this site comes from a master list of repositories, stored in a [publicly available JSON file](https://github.com/SwiftPackageIndex/MasterPackageList/blob/master/packages.json). To add a package to the index, add the URL of the repository to that file.
+Adding a new package to the Swift Package Index is straightforward. Every package indexed by this site comes from a master list of repositories, stored in a [publicly available JSON file](https://github.com/SwiftPackageIndex/MasterPackageList/blob/main/packages.json). To add a package to the index, add the URL of the repository to that file.
```

Comment:
Ah, good catch! And I just noticed that it also still says `MasterPackageList`. That should be `PackageList` instead.

If you've got a chance to update that as well that'd be great. Otherwise I'll merge and fix it up later today.

Reply:
Got it! Didn't catch it because MasterPackageIndex still redirects automatically.

Reply:
Ah yes, you're right. It _would_ have worked actually. Better this way though - thanks for following up!